### ISSUE TITLE
equationsToMatrix: wrong result on simple system

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#961]
With SymPy 1.4rc2:
```
>> eqn = a*(x + 2*y) + b*y == 0
eqn = (sym) a⋅(x + 2⋅y) + b⋅y = 0

>> equationsToMatrix(eqn, [x; y])   # WRONG
ans = (sym) [a  b]  (1×2 matrix)

>> expand(eqn)
ans = (sym) a⋅x + 2⋅a⋅y + b⋅y = 0

>> equationsToMatrix(ans, [x; y])    # much better
ans = (sym) [a  2⋅a + b]  (1×2 matrix)
```
Probably upstream issue...

(We had a lot of problems with this upstream: unfort. still not fixed).

**Comment by cbm755** [comment]
upstream: https://github.com/sympy/sympy/issues/16577

If we can't get this fixed for 1.4, Symbolic should keep using our own implementation for another SymPy cycle.

**Comment by cbm755** [comment]
Awesome, my upstream fix was merged for 1.4
